platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/action/StartMaintenanceAction.java

```diff
@@ -0,0 +1,57 @@
+package com.softicar.platform.core.module.maintenance.action;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.io.resource.IResource;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreImages;
+import com.softicar.platform.core.module.CoreRoles;
+import com.softicar.platform.core.module.maintenance.AGMaintenance;
+import com.softicar.platform.core.module.maintenance.MaintenancePredicates;
+import com.softicar.platform.core.module.maintenance.status.AGMaintenanceStatusEnum;
+import com.softicar.platform.db.core.transaction.DbTransaction;
+import com.softicar.platform.emf.action.IEmfManagementAction;
+import com.softicar.platform.emf.authorization.role.IEmfRole;
+import com.softicar.platform.emf.predicate.IEmfPredicate;
+
+public class StartMaintenanceAction implements IEmfManagementAction<AGMaintenance> {
+
+	@Override
+	public IEmfPredicate<AGMaintenance> getPrecondition() {
+
+		return MaintenancePredicates.PENDING//
+			.and(MaintenancePredicates.NEXT_IN_LINE)
+			.and(MaintenancePredicates.NO_MAINTENANCE_ACTIVE);
```

Comment:
- IMO `NEXT_IN_LINE` und `NO_MAINTENANCE_ACTIVE` should not be added here
- instead, NEXT_IN_LINE should trigger a confirmation in `handleClick`, that they are about to start a maintenance window which is not next in line
- NO_MAINTENANCE_ACTIVE should trigger an exception in `handleClick`